activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/converters/ActivityCancelledEventConverter.java

```diff
@@ -20,20 +20,25 @@ public String getSupportedEvent() {
-                                                                                                                    cloudBPMNActivityCancelledEvent.getServiceFullName(),
-                                                                                                                    cloudBPMNActivityCancelledEvent.getServiceName(),
-                                                                                                                    cloudBPMNActivityCancelledEvent.getServiceType(),
-                                                                                                                    cloudBPMNActivityCancelledEvent.getServiceVersion(),
-                                                                                                                    cloudBPMNActivityCancelledEvent.getEntity(),
-                                                                                                                    cloudBPMNActivityCancelledEvent.getCause());
-        activityCancelledAuditEventEntity.setEntityId(cloudBPMNActivityCancelledEvent.getProcessInstanceId());
-        activityCancelledAuditEventEntity.setProcessDefinitionId(cloudBPMNActivityCancelledEvent.getProcessDefinitionId());
-        activityCancelledAuditEventEntity.setProcessInstanceId(cloudBPMNActivityCancelledEvent.getProcessInstanceId());
-        return activityCancelledAuditEventEntity;
+        ActivityCancelledAuditEventEntity auditEventEntity = new ActivityCancelledAuditEventEntity(cloudBPMNActivityCancelledEvent.getId(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getTimestamp(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getAppName(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getAppVersion(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getServiceFullName(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getServiceName(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getServiceType(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getServiceVersion(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getEntity(),
+                                                                                                   cloudBPMNActivityCancelledEvent.getCause());
+        
+        auditEventEntity.setBaseProcessData(cloudBPMNActivityCancelledEvent.getEntityId(),
```

Comment:
Instead of having this method taking several parameters I'd rather pass `AuditEventEntity` as parameter. In addition, it's probably better to use an external class to this logic so you don't need to modify the model and you can use the same approach for `convertToAPI` method without the need to modify classes in another git repository. I'll push an example of what I have in mind for this class, then you can apply the same approach of other converters if it looks ok for you.